docs/guide/alerts-guide.md

```diff
@@ -37,21 +37,21 @@ Default Alerts for a Monitoring App:
 - **Heartbeat Alert** if the **Agent stops sending data to the Monitoring App**
 - **Metric Anomaly Alert** for misbehaving Nodes
 
-> In this example which is a Elasticsearch Monitoring App, the Java usage threshold and Elasticsearch Node Anomalies are integration-specific default Alerts.
+In this example which is an Elasticsearch Monitoring App, the Java usage threshold and Elasticsearch Node Anomalies are integration-specific default Alerts.
 
 ![Sematext Cloud System Generated Alerts](https://sematext.com/docs/images/guide/alerts-and-events/system-generated-alerts.png "Sematext Cloud System Generated Alerts")
 
-The [Alert Rules](https://apps.sematext.com/ui/events/alerts/rules) page is where you will find all default, and created, Alerts for all Apps you'ce created. Here you can toggle, edit or delete any existing Alerts.
+The [Alert Rules](https://apps.sematext.com/ui/events/alerts/rules) page is where you will find all default and created, Alerts for all Apps you've created. Here you can toggle, edit or delete any existing Alerts.
```

Comment:
"page is where you will find all default and created, Alerts for all Apps you've created" sounds strange, something is a little extra there...

Reply:
Reworded this to sounds better.